specification/eventgrid/data-plane/Microsoft.AVS/preview/2023-09-01-preview/PrivateCloud.json

```diff
@@ -0,0 +1,216 @@
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.PrivateCloudFailed event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSPrivateCloudEventProperties"
+        }
+      ],
+      "properties": {
+        "failureMessage": {
+          "description": "Failure reason of an event.",
+          "type": "string"
+        }
+      }
+    },
+    "AVSClusterCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ClusterCreated event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSClusterEventProperties"
+        }
+      ],
+      "properties": {}
+    },
+    "AVSClusterDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ClusterDeleted event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSClusterEventProperties"
+        }
+      ],
+      "properties": {}
+    },
+    "AVSClusterUpdatingEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ClusterUpdating event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSClusterEventProperties"
+        }
+      ],
+      "properties": {}
+    },
+    "AVSClusterUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ClusterUpdated event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSClusterEventProperties"
+        }
+      ],
+      "properties": {}
+    },
+    "AVSClusterFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ClusterFailed event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSClusterEventProperties"
+        }
+      ],
+      "properties": {
+        "failureMessage": {
+          "description": "Failure reason of an event.",
+          "type": "string"
+        }
+      }
+    },
+    "AVSScriptExecutionStartedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ScriptExecutionStarted event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSScriptExecutionEventProperties"
+        }
+      ],
+      "properties": {}
+    },
+    "AVSScriptExecutionFinishedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.ScriptExecutionFinished event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSScriptExecutionEventProperties"
+        }
+      ],
+      "properties": {
+        "namedOutputs": {
+          "description": "Named outputs of completed execution, if any.",
+          "type": "object",
+          "items": {
```

Comment:
Josh, you mentioned there's some decoration we can apply to make the property generate as `Dictionary<string,string>`?